opm/autodiff/SimulatorBase.hpp

```diff
@@ -0,0 +1,195 @@
+/*
+  Copyright 2013 SINTEF ICT, Applied Mathematics.
```

Comment:
It seems to me that you have made sufficient changes to warrant adding your own copyright to this. I assume everybody treats this in a reasonable and sensible manner...